### PR TITLE
Adding aurelia-formio library

### PR DESCRIPTION
**Pull request by travist**

### CODE REVIEW

**Comment by voidberg** [comment]
Thanks! Please update the PR and remove the trailing space as our tests are failing.

File: README.md [hunk 1]
```diff
@@ -201,6 +201,7 @@ A curated list of amazingly awesome Aurelia libraries, resources and shiny thing
 * [aurelia-keyboard-plugin](https://github.com/Novatrox/aurelia-keyboard-plugin) * Simple keyboard plugin for aurelia
 * [aurelia-ui-framework](https://github.com/adarshpastakia/aurelia-ui-framework) * A bespoke UI Framework built on Aurelia for desktop business application
 * [aurelia-tags-input](https://github.com/TGMorais/aurelia-tags-input) * An aurelia tags input plugin
+* [aurelia-formio](https://github.com/formio/aurelia-formio) * A dynamic JSON Form Renderer and Form Builder - [See Example](https://formio.github.io/aurelia-formio/)
```
Comment (voidberg) [inline README.md]: The trailing spaces are breaking our CI.

**Comment by travist** [comment]
Sorry about that. Should be fixed now.

**Comment by voidberg** [comment]
Thanks!

**Pull request merged**